mne/gui/_coreg.py

```diff
@@ -648,20 +648,21 @@ def _add_head_shape_points(self):
     def _add_eeg_channels(self):
         if self._eeg_channels:
             eeg = ["original"]
-            picks = pick_types(self._info, eeg=(len(eeg) > 0))
+            picks = pick_types(self._info, eeg=(len(eeg) > 0), fnirs=True)
             if len(picks) > 0:
-                eeg_actors = _plot_sensors(
+                actors = _plot_sensors(
                     self._renderer, self._info, self._to_cf_t, picks,
-                    meg=False, eeg=eeg, fnirs=False, warn_meg=False,
-                    head_surf=self._head_geo, units='m',
+                    meg=False, eeg=eeg, fnirs=["sources", "detectors"],
+                    warn_meg=False, head_surf=self._head_geo, units='m',
                     sensor_opacity=self._defaults["sensor_opacity"],
                     orient_glyphs=self._orient_glyphs, surf=self._head_geo)
-                eeg_actors = eeg_actors["eeg"]
+                sens_actors = actors["eeg"]
+                sens_actors.append(actors["fnirs"])
```

Comment:
```suggestion
                sens_actors.extend(actors["fnirs"])
```

This could deal with the issues with `SetVisibillity()`